File: AgentApp/Frontend/rent-ng-app/src/app/model/carClass.ts
/* tslint:disable:whitespace */
export class CarClass{
  id: number;
  isRemoved: boolean;
  className: string;

  constructor(id: number, isRemoved: boolean, name: string){
    this.id=id;
    this.isRemoved=isRemoved;
    this.className=name;
  }
}

File: AgentApp/Backend/src/main/java/rent/app/model/CarClass.java
package rent.app.model;

import lombok.Data;

import javax.persistence.*;
import javax.validation.constraints.Max;
import javax.validation.constraints.Min;

@Data
@Entity
@Table(name="CarClass")
public class CarClass {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column
    private String className;
    @Column
    private boolean isRemoved;

}
